lms/djangoapps/verify_student/views.py

```diff
@@ -1009,6 +1010,16 @@ def results_callback(request):
         log.debug("Approving verification for %s", receipt_id)
         attempt.approve()
         status = "approved"
+        try:
+            checkpoints = VerificationCheckpoint.objects.filter(photo_verification=attempt)[0]
+            course_key = checkpoints.course_id
+            credit_requirement = CreditRequirement.get_course_requirement(
+                course_key, "reverification", checkpoints.checkpoint_name
+            )
+            CreditRequirementStatus.add_requirement_status(attempt.user.username, credit_requirement, "satisfied")
+        except Exception as exp:
```

Comment:
Why generic exception? If its necessary then add pylint disable text to avoid pylint error:
```
except Exception as exp:  # pylint: disable=broad-except
```